model/model/v1alpha/model.proto

```diff
@@ -1724,8 +1724,9 @@ message UndeployOrganizationModelAdminResponse {}
 message ModelRun {
   // Model Run UUID.
   string uid = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Model UUID.
-  string model_uid = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // 2 is reserved for the model UUID. Public resources should be identified by
+  // their ID and parent resource.
+  reserved 2;
```

Comment:
⚠️ TODO: Do these breaking affect only the backend, or are these fields used in the frontend?

- [ ] `modelUid`
- [ ] `pipelineUid`
- [ ] `namespaceId`